app/src/main/java/protect/card_locker/LoyaltyCardViewActivity.java

```diff
@@ -1129,11 +1162,21 @@ private void setFullscreen(boolean enabled) {
             binding.container.setVisibility(View.GONE);
             binding.fullscreenLayout.setVisibility(View.VISIBLE);
 
+            if (format.isSquare()) {
+                binding.setWidthLayout.setVisibility(View.GONE);
+            }
+            else {
+                binding.setWidthLayout.setVisibility(View.VISIBLE);
+            }
```

Comment:
We can make this a lot smaller :)

```suggestion
            // Only show width slider if the barcode isn't square (square barcodes will resize height and width together)
            binding.setWidthLayout.setVisibility(format.isSquare() ? View.GONE : View.VISIBLE);
```

Reply:
And also in 2226e3c0
Many thanks for reviewing :)